runtime/queries/json/highlights.scm

```diff
@@ -4,6 +4,6 @@
 (string) @string
 
 (object
-  "{" @escape
+  "{" @constant.character.escape
   (_)
-  "}" @escape)
+  "}" @constant.character.escape)
```

Comment:
Should these be `@punctuation.bracket`?

Reply:
Yep. It's really barebones too, let's add number and boolean highlighting 

https://github.com/nvim-treesitter/nvim-treesitter/blob/bd1399105026c7b067dc197b31ccdb2602700a86/queries/json/highlights.scm